python/py-precis-i18n/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        byllyfish precis_i18n 1.0.4 v
```

Comment:
no need for GitHub here, just take the port from PyPI which we prefer for Python ports

Reply:
Switched to Pypi now.